components/selectors/matching.rs

```diff
@@ -328,8 +328,7 @@ where
         debug_assert_eq!(next_sequence, Combinator::PseudoElement);
     }
 
-    let result =
-        matches_complex_selector_internal(iter, element, context, Rightmost::Yes);
+    let result = matches_complex_selector_internal(iter, element, context, Rightmost::Yes);
```

Comment:
Instead of fixing the style in the selectors crate, I think we should add an exception and fix the style upstream.